src/db/migrations/migration1.js

```diff
@@ -0,0 +1,31 @@
+const _ = require('lodash');
+const fs = require('fs-extra');
+const { getLogger } = require('../../utils/logger');
+const Utils = require('../../utils');
+
+async function addLanguageField(db) {
+  try {
+    await db.update({ language: { $exists: false } }, { $set: { language: 'en-US' } }, { multi: true });
+  } catch (err) {
+    throw Error(`DB update Error: ${err.message}`);
+  }
+}
+
+
+async function migration1(db) {
+  const migrationTrackPath = `${__dirname}/../migrations.dat`;
+  try {
+    let lastMigrate = Number(await fs.readFileSync(migrationTrackPath).toString().split('=')[1].trim());
+    if (lastMigrate < 1) {
+      await addLanguageField(db.Oracles);
+      await addLanguageField(db.Topics);
+      lastMigrate++;
+      getLogger().info(`Success migration${lastMigrate}`);
+    }
+    await fs.outputFileSync(migrationTrackPath, `LAST_MIGRATION=${lastMigrate}\n`);
```

Comment:
this logic of writing to `migrations.dat` should be outside of any migration file. move this to `applyMigrations()` since it is generic logic and not specific to the a single migration. otherwise you would have to keep duplicating this code in every migration.